src/components/Navigation/Navigation.stories.js

```diff
@@ -0,0 +1,10 @@
+import Navigation from './Navigation'
+
+export default {
+  title: 'Navigation',
+  component: Navigation,
+}
+
+const Template = args => <Navigation />
```

Comment:
I think you can just use brackets instead of the args, because you don't have any args. Like this:
const Template = () => <Navigation />